pyproject.toml

```diff
@@ -27,13 +27,6 @@ __all__ = ["__version__"]
 __version__ = "{version}"
 """
 
-[tool.pytest.ini_options]
-addopts = "--black --flake8 --mypy --ignore-glob=*/version.py"
-flake8-ignore = ["E133", "E203", "E226", "E228", "N802", "N803", "N806", "N812", "N813", "N815", "N816", "W503"]
-flake8-max-line-length = 110
-flake8-max-doc-length = 79
-asyncio_mode = "auto"
-
 [tool.mypy]
```

Comment:
This section may be removed as well since ts_pre_commit_conf will generate a mypy.ini file.